settings.gradle

```diff
@@ -1 +1 @@
-include ':TMessagesProj'
+include :'TMessagesProj'
```

Comment:
startup failed:
settings file 'C:\Telegram-master\settings.gradle': 1: Statement labels may not be used in build scripts.
In case you tried to configure a property named 'include', replace ':' with '=' or ' ', otherwise it will not have the desired effect.
 @ line 1, column 10.
   include :'TMessagesProj'